guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc

```diff
@@ -13,31 +13,31 @@ For more information, see {InstallingSmartProxyDocURL}configuring-remote-executi
 * The host is able to communicate with its {SmartProxy} over HTTPS.
 
 .Procedure
-. Install the `katello-pull-transport-migrate` package on your host:
+* Install the `katello-pull-transport-migrate` package on your host:
 ifdef::client-content-dnf[]
-* On {EL} 8 and {EL} 9 hosts:
+** On {EL} 8 and {EL} 9 hosts:
```

Comment:
Unrelated but since we're already on the topic of improving the procedure, might as well.
```suggestion
** On {EL} 9 and {EL} 8 hosts:
```